osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -195,7 +197,10 @@ private void load(BeatmapManager beatmaps, AudioManager audio, DialogOverlay dia
                 this.beatmaps = beatmaps;
 
             if (osu != null)
+            {
+                this.osu = osu;

```

Comment:
What do you need this for? If it's later for accessing the ruleset then isn't it bound to `Ruleset` and thus accessable anyways?